Mathlib/Init/Set.lean

```diff
@@ -39,11 +39,12 @@ p
 
 namespace Set
 
-protected def mem (a : α) (s : Set α) :=
+/-- Membership in a set -/
```

Comment:
Since you added this doctring, you should be able to remove the nolint: https://github.com/leanprover-community/mathlib4/blob/e0335f963230a21b1de9dbb0596e9cac88c05961/scripts/nolints.json#L289

Reply:
we don't have the bot to remove nolints?

Reply:
I don't see any bot committing to that file: https://github.com/leanprover-community/mathlib4/commits/master/scripts/nolints.json

But maybe there's something that I don't know about?

Reply:
We had such a bot for mathlib3. We better get it set up here.